core/css.js

```diff
@@ -142,12 +142,13 @@ Blockly.Css.CONTENT = [
   '  background-color: #fff;',
   '  outline: none;',
   '  overflow: hidden;',  /* IE overflows by default. */
+  '  display: block;', 
   '}',
 
   '.blocklyWidgetDiv {',
   '  display: none;',
   '  position: absolute;',
-  '  z-index: 999;',
+  '  z-index: 99999 !important;',
```

Comment:
But will it work without it? If so, I'd recommend to remove it as you never know what the requirements of the developer might be or how it might interact with other frameworks.